NET01.2Solution/NET01.2Task/DiagonalMatrix.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NET01._2Task
```

Comment:
Follow PascalCase for namespace naming https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces.